<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1097</width>
    <height>745</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Record IR</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_5">
        <item>
         <widget class="QPushButton" name="startStreamBut">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="text">
           <string>Start IR Camera Feed</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ffcBut">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Perform FFC</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <widget class="QPushButton" name="cmIronBut">
          <property name="text">
           <string>Ironblack</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cmGrayBut">
          <property name="text">
           <string>Grayscale</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="cmRainBut">
          <property name="text">
           <string>Rainbow</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Set Gain State:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboGain">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>HIGH</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>LOW</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>AUTO</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_6">
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Set FFC Mode:</string>
          </property>
          <property name="alignment">
           <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="comboFFCmode">
          <property name="enabled">
           <bool>true</bool>
          </property>
          <item>
           <property name="text">
            <string>AUTO</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>MANUAL</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>EXTERNAL</string>
           </property>
          </item>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Begin Filename With:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="lineEdit">
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="filePathBut">
        <property name="text">
         <string>Select File Path</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QPushButton" name="startRec">
            <property name="font">
             <font>
              <weight>75</weight>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Start / Stop Recording Data</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="stopRec">
            <property name="text">
             <string>Stop Recording/Save</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="recLabel">
          <property name="font">
           <font>
            <italic>true</italic>
           </font>
          </property>
          <property name="text">
           <string>Not Recording</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QPushButton" name="displayC">
          <property name="text">
           <string>Display Celsius</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="displayF">
          <property name="text">
           <string>Display Fahrenheit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="timeStatus">
        <property name="text">
         <string>Time/Date</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="cursorTempLabel">
        <property name="text">
         <string>Cursor Temp (On Mouse Click)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="maxTempLabel">
        <property name="text">
         <string>Max Temperature</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="minTempLabel">
        <property name="text">
         <string>Min Temperature</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="storageLabel">
        <property name="text">
         <string>Storage</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="runPost">
        <property name="text">
         <string>Run Post Processing Script</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="printShutterBut">
        <property name="text">
         <string>Print Shutter Info to Terminal</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Command Log:</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0" colspan="3">
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_7">
        <item>
         <widget class="QLabel" name="labelForLocation">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>0</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>175</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>File Save Location:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="filePathDisp">
          <property name="text">
           <string>Please Specify Using the Select File Path Button to the Right</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="plainTextEdit">
        <property name="plainText">
         <string>Notes/Comment Area</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0" rowspan="2">
     <widget class="QLabel" name="displayFrame">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="minimumSize">
       <size>
        <width>640</width>
        <height>480</height>
       </size>
      </property>
      <property name="text">
       <string>Click &quot;Start IR Camera Feed&quot; once camera ready.</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPlainTextEdit" name="history">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Expanding" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>87</height>
       </size>
      </property>
     </widget>
    </item>
    <item row="0" column="1" rowspan="2">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
